src/transform3d.rs

```diff
@@ -177,6 +177,18 @@ where T: Copy + Clone +
         )
     }
 
+    /// Check whether shapes on the XY plane with Z pointing towards the
+    /// screen transformed by this matrix would be facing back.
+    pub fn is_backface_visible(&self) -> bool {
+        // inverse().m33 < 0;
```

Comment:
could you provide the links/sources onto why you think `inverse().m33 < 0` is our target condition for backface visibility?